src/service/iot_config.proto

```diff
@@ -365,6 +365,17 @@ message gateway_region_params_res_v1 {
   bytes signature = 4;
 }
 
+message load_region_req_v1 {
+  region region = 1;
+  blockchain_region_params_v1 params = 2;
```

Comment:
We have to support being able to update the region params separate from the hex_indexes since they update on very different schedules. The proto can support it through the optional/default values for either params or hex_indexes but wanted to make sure you got the requirement for the implementation